ons_alpha/jinja2/templates/pages/bulletins/bulletin_page.html

```diff
@@ -1,17 +1,10 @@
-{% extends "templates/base_page.html" %}
+{% extends "base_page.html" %}
 
 {% block main %}
     {{ table_of_contents(page, "body", {"title": "Table of contents"}) }}
-
     {% include_block page.updates %}
-
     {% include_block page.body %}
-
+    <div class="previous-releases">
```

Comment:
This doesn't take into account whether it this bulletin page is the latest, or not.

I suggest adding an `is_latest` property on `BulletinPage` that does the following check:

```python
class BulletinPage(...):
    ...

    @property
    def is_latest(self):
        latest_id = ReleasePage.objects.sibling_of(self).live().order_by('-release_date').values_list('id', flat=True).first()
         return self.pk == latest_id
```

ref: https://docs.wagtail.org/en/stable/reference/pages/queryset_reference.html#wagtail.query.PageQuerySet.sibling_of

Reply:
this is also missing the "view latest" box for older bulletins

Reply:
You added the property on the model, and now it needs handling in the template